content/index.md

```diff
@@ -1,18 +1,22 @@
 ---
-title: OSRA Infrastructure Project
+title: Open robotics infrastructure project
```

Comment:
```suggestion
title: Open Robotics infrastructure project
```

Reply:
Shouldn't be this "Open Robotics Infrastructure project"? Refering to the comment below